.circleci/config.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     <<: *node_container
     steps:
       - checkout
+      - run: rm .npmrc # npmrc only used for publishing
```

Comment:
The workspace only includes the node_modules directory, so `.npmrc` deletion is not persisted.